subsys/bluetooth/controller/ll_sw/nordic/hal/nrf5/radio/radio.h

```diff
@@ -5,6 +5,48 @@
  * SPDX-License-Identifier: Apache-2.0
  */
 
+/* Set of macros related with Radio packet configuration flags */
+#define RADIO_PKT_CONF_
```

Comment:
Is it normal to add this sort of define for the radio headers?

Reply:
This is a first time, but these macros are used outside of the `radio.c` so I think there is no better place to put them.

Reply:
I was more thinking about the explicit empty macro `RADIO_PKT_CONF_`. What purpose does it serve?

Reply:
ah this is a leftover... sorry I didn't spot that.